src/azure-cli/azure/cli/command_modules/vm/custom.py

```diff
@@ -2933,3 +2933,23 @@ def execute_query_for_vm(cmd, client, resource_group_name, vm_name, analytics_qu
                        'Please check the status of log analytics workpsace.')
     return client.query(workspace, QueryBody(query=analytics_query, timespan=timespan))
 # endregion
+
+
+# disk encryption set
+def create_disk_encryption_set(cmd, client, resource_group_name, disk_encryption_set_name,
+                               key_url, source_vault, location=None, tags=None):
+    from msrestazure.tools import resource_id, is_valid_resource_id
```

Comment:
just curious, why not import at global level? this is quite common methods

Reply:
> 
> 
> just curious, why not import at global level? this is quite common methods

Follow the convention of existing functions. We can import them at global level, which saves many lines of code.